examples/dmm/dmm.py

```diff
@@ -150,7 +150,7 @@ def __init__(self, input_dim=88, z_dim=100, emission_dim=100,
 
         # if we're using normalizing flows, instantiate those too
         self.iafs = [InverseAutoregressiveFlow(AutoRegressiveNN(z_dim, [iaf_dim])) for _ in range(num_iafs)]
-        self.iafs_modules = nn.ModuleList([iaf.module for iaf in self.iafs])
+        self.iafs_modules = nn.ModuleList([self.iafs])
```

Comment:
Remove the square brackets, since `self.iafs` is already a list.

Reply:
Ah sorry, yes of course! :)